<?xml version="1.0"?>
<launch>
  <node name="plan2ctrl" pkg="motion_planner" type="motion_planner_pouring_tool" output="screen" respawn="false">
  	<param name="action_mode" value="vision"/>
  	<param name="simulation" value="false" />
  	<param name="world_frame" value="/world_frame" />
  	<param name="model_dt" value="0.01" />
  	<param name="reachingThreshold" value="0.02" />
  	<param name="orientationThreshold" value="0.2" />
	<param name="model_base_path" value="$(find motion_planner)../epfl-task-models/PouringModels" />
		  <param name="wait_for_force" value="false" />
  </node>
</launch>
